editor/components/post-featured-image/index.js

```diff
@@ -22,9 +23,26 @@ import MediaUpload from '../media-upload';
 const DEFAULT_SET_FEATURE_IMAGE_LABEL = __( 'Set featured image' );
 const DEFAULT_REMOVE_FEATURE_IMAGE_LABEL = __( 'Remove image' );
 
-function PostFeaturedImage( { featuredImageId, onUpdateImage, onRemoveImage, media, postType } ) {
+function PostFeaturedImage( { currentPostId, featuredImageId, onUpdateImage, onRemoveImage, media, postType } ) {
 	const postLabel = get( postType, [ 'labels' ], {} );
 
+	let mediaWidth,
+		mediaHeight,
+		mediaSourceUrl;
+	if ( media ) {
+		let mediaSize = 'post-thumbnail';
+		mediaSize = applyFilters( 'editor.PostFeaturedImage.imageSize', mediaSize, media.id, currentPostId );
```

Comment:
Seems like this could just be `const mediaSize = applyFilters( 'editor.PostFeaturedImage.imageSize', 'post-thumbnail', media.id, currentPostId );`, then you can ditch the previous line.

Reply:
246b3faefe622199dc3733620560e058d6a34810